Library/Homebrew/utils/tty.rb

```diff
@@ -0,0 +1,51 @@
+module Tty
+  module_function
+
+  def strip_ansi(string)
+    string.gsub(/\033\[\d+(;\d+)*m/, "")
+  end
+
+  def width
+    `/usr/bin/tput cols`.strip.to_i
+  end
+
+  def truncate(string)
+    (w = width).zero? ? string.to_s : string.to_s[0, w - 4]
+  end
+
+  COLOR_CODES = {
+                  red:     31,
+                  green:   32,
+                  yellow:  33,
+                  blue:    34,
+                  magenta: 35,
+                  cyan:    36,
+                  default: 39,
```

Comment:
Tiny nit: just indent these at four spaces in and use hash rockets for now (we'll do a mass update 🔜)